test/helpers/kubectl.go

```diff
@@ -2911,9 +2911,11 @@ func (kub *Kubectl) CiliumExecContext(ctx context.Context, pod string, cmd strin
 	for i := 0; i < limitTimes; i++ {
 		res = execute()
 		switch res.GetExitCode() {
-		case -1, 126:
-			// Retry.
```

Comment:
Previously we didn't log anything for these error codes. With this change we now log a warning in the `default` clause below. I'm not sure how frequent these errors are, but given the comment above they might be? Maybe we should keep the special case for these error codes to avoid spamming the logs two much.

Reply:
@tklauser I've returned the case that catches '-1' and '126' return codes and added explanation in the comments why this exists.